### ISSUE TITLE
Just wondering: Should there be a .d.ts file?

### ISSUE THREAD

**Issue by kubycsolutions** [kartik1998/phonetics#10]
If I use require rather than import,  `const Phonetics = require('phonetics');`, my Typescript compiles and runs happily.

If I try to use the new `import from` syntax, Typescript complains:
```
src/new/episodesdb.ts:32:23 - error TS7016: Could not find a declaration file for module 'phonetics'. 'C:/Users/keshlam/jovo/new-sounds-on-demand/node_modules/phonetics/build/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/phonetics` if it exists or add a new declaration (.d.ts) file containing `declare module 'phonetics';`

32 import Phonetics from 'phonetics'
                         ~~~~~~~~~~~
```
I believe I get a similar error if I try `import doubleMetaphone from 'phonetics'`.

The workaround, obviously, is to use the require syntax. But if we could support both, that would probably be a Good Thing.

**Comment by kartik1998** [comment]
Hey @kubycsolutions thanks for highlighting this. I tried to fix this by adding a declaration file but it didn't work as i expected. I'll fix it and let you know.

**Comment by kartik1998** [comment]
@kubycsolutions this has been resolved with v1.0.7 you can import the module like `import * as Phonetics from 'phonetics'`
